src/components/popover-container/popover-container.component.tsx

```diff
@@ -168,6 +169,33 @@ export const PopoverContainer = ({
       setTimeout(() => closeButtonRef.current?.focus(), 0);
   }, [isOpen]);
 
+  const handleEscKey = useCallback(
+    (ev) => {
+      const eventIsFromSelectInput = Events.composedPath(ev).find((p) => {
```

Comment:
**comment**:  not a fan of the name `p` for this variable, although it looks like you took it from `submenu` where we're doing something very similar. Short variable names are almost always bad (counter variables in loops being the main exception), and I can't even see what `p` stands for (not `path` I hope because `p` here isn't the path, it's an element that's part of the path. For that reason I'd much rather call it something like `element`, or `target` (since technically its type is `EventTarget`).

Reply:
I can certainly change the variable to something a bit descriptive and I most certainly stole the code from `submenu`. I'll get `submenu` updated too but in a separate PR.